doc/dev_guide/building_from_source.rst

```diff
@@ -5,279 +5,226 @@
-       * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
-       * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
+    *   Python and modules.
 
-   To install all required dependencies, follow the instructions for your OS:
+        Python interpreter is not necessary for building Tarantool itself, unless you
+        intend to use the “Run the test suite". For all platforms, this is python
+        version 3.x. You need the following Python modules:
 
-   * For Debian/Ubuntu, say:
+        *   pyyaml
+        *   gevent
+        *   six
 
-     .. code-block:: console
+    To install all required dependencies, follow the instructions for your OS:
 
-        $ apt install -y build-essential cmake make coreutils sed \
-              autoconf automake libtool zlib1g-dev \
-              libreadline-dev libncurses5-dev libssl-dev \
-              libunwind-dev libicu-dev \
-              python python-pip python-setuptools python-dev \
-              python-msgpack python-yaml python-argparse python-six python-gevent
+    *   For Debian/Ubuntu, say:
 
-   * For RHEL/CentOS (versions under 8)/Fedora, say:
+        ..  code-block:: console
 
-     .. code-block:: console
+            $ apt-get install -y \
+              git build-essential cmake make coreutils autoconf automake libtool sed \
+              python3 python3-yaml python3-six python3-gevent
 
-         $ yum install -y gcc gcc-c++ cmake make coreutils sed \
-               autoconf automake libtool zlib-devel \
-               readline-devel ncurses-devel openssl-devel \
-               libunwind-devel libicu-devel \
-               python python-pip python-setuptools python-devel \
-               python-msgpack python-yaml python-argparse python-six python-gevent
+    *   For RHEL/CentOS (versions under 8)/Fedora, say:
 
-   * For CentOS 8, say:
+        ..  code-block:: console
 
-     .. code-block:: console
+            $ yum install -y \
+              git perl gcc cmake make gcc-c++ libstdc++-static autoconf automake libtool \
+              python3-yaml python3-six python3-gevent
 
-         $ yum install epel-release
-         $ curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | sudo bash
-         $ yum install -y gcc gcc-c++ cmake make coreutils sed \
-               autoconf automake libtool zlib-devel \
-               readline-devel ncurses-devel openssl-devel \
-               libunwind-devel libicu-devel \
-               python2 python2-pip python2-setuptools python2-devel \
-               python2-yaml python2-six
+    *   For CentOS 8, say:
 
-   * For Mac OS X (instructions below are for OS X El Capitan):
+        ..  code-block:: console
 
-     If you're using Homebrew as your package manager, say:
+            $ yum install epel-release
+            $ curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | sudo bash
```

Comment:
The commands below and above require superuser privileges as well. So it is assumed that it is run from `root`: `sudo` is unnecessary here and may confuse a reader.

Reply:
There still no consistency in using `sudo`: some `yum` commands are written with `sudo`, but some without. There is also no consistency in using `-y`.